hadoop-hdds/docs/content/feature/OM-HA.md

```diff
@@ -23,13 +23,13 @@ summary: HA setup for Ozone Manager to avoid any single point of failure.
   limitations under the License.
 -->
 
-Ozone has two metadata-manager nodes (*Ozone Manager* for key space management and *Storage Container Management* for block space management) and multiple storage nodes (Datanode). Data is replicated between Datanodes with the help of RAFT consensus algorithm.
+Ozone has two metadata-manager nodes (*Ozone Manager* for key space management and *Storage Container Manager* for block space management) and multiple storage nodes (Datanode). Data is replicated between Datanodes with the help of RAFT consensus algorithm.
```

Comment:
Can you please rebase your changes on top of the latest master, looks like these changes have been addressed as part of PR: https://github.com/apache/ozone/pull/6134.